src/utils/type-transform/BackupFormat/JSON/version-2.ts

```diff
@@ -65,6 +65,19 @@ export interface BackupJSONFileVersion2 {
         recipientGroups: string[] // Array<GroupIdentifier.toText()>
         foundAt: number // Unix timestamp
     }>
+    wallets: Array<{
```

Comment:
I think you should leave it as `wallets?: ...` and make a note that this property is new and not all v2 backup has it.

Reply:
I filled this field up before upgrading the backup, so all v2 backup will got this.